src/neuvector/chart/templates/uds-package.yaml

```diff
@@ -4,6 +4,25 @@ metadata:
   name: neuvector
   namespace: {{ .Release.Namespace }}
 spec:
+  sso:
+    - name: Neuvector
+      clientId: uds-core-admin-neuvector
+      redirectUris:
+        - "https://neuvector.admin.{{ .Values.domain }}/openId_auth"
+      secretName: neuvector-secret
```

Comment:
Not a real comment, but I think this makes sense to use instead of `neuvector-init` because it allows us to continue using `secret.enabled` if we needed it for anything else. Kinda weird that the controller even mounts this since its not created anywhere in the chart.

Reply:
Oh, good catch.  This works but I don't think I consciously chose that secret name.